README.md

```diff
@@ -41,6 +41,46 @@ The Image Tool provides three functions within the main script:
 
 Use the [Quick Start](site/quickstart.md) guide to create a Linux based WebLogic Docker image.
 
+## Building From Source
+
+The Image Tool installer is available for download on the [Releases](releases) page.  If you wish
```

Comment:
on the [Releases](releases) page. -> not a functioning hyperlink
If you wish -> If you want
(wish (n) (v) Avoid. Use want instead.)